Scripts/Source/minai_Sex.psc

```diff
@@ -278,6 +278,7 @@ EndFunction
 
 Function ActionResponse(actor akTarget, actor akSpeaker, string sayLine, actor[] actorsFromFormList, bool bPlayerInScene)
   actor Player = game.GetPlayer()
+  actor[] actorsFromFormList = AiAgentFunctions.findAllNearbyAgents()
```

Comment:
This will break the mantella integration for this. We want to call AiAgentFunctions.findAllNearbyAgents() from the command dispatcher, not here.

Reply:
oops